lib/ansible/plugins/lookup/password.py

```diff
@@ -28,11 +28,11 @@
          required: True
       encrypt:
         description:
-           - Whether the user requests that this password is returned encrypted or in plain text.
-           - Note that the password is always stored as plain text.
+           - Which hash scheme to encrypt the returning password, should be one hash scheme from C(passlib.hash).
+           - If not provided, the password will be returned in plain text.
+           - Note that the password is always stored as plain text, only the returning password is encrypted.
            - Encrypt also forces saving the salt value for idempotence.
-        type: boolean
-        default: True
+        default: None
```

Comment:
you should make note on how the parameter changed (used to be boolean) and in which version

Reply:
Take a look of this:
https://github.com/ansible/ansible/blob/stable-2.0/lib/ansible/utils/encrypt.py#L71
Looks like even back to ansible 2.0, the `do_encrypt`  treat `encrypt` as the name of encryption schema. So the bool type in the doc is always wrong in the history, should we note a wrong history?

Reply:
yeah, even if just `- for a long time this was incorrectly labeled as a boolean`

Reply:
then backport